src/promptflow-devkit/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -1016,7 +1019,38 @@ def _merge_signature(extracted, signature_overrides):
         return signature
 
     @staticmethod
-    def _infer_signature(
+    def _infer_signature(entry: Union[Callable, FlexFlow, Flow, Prompty]):
+        if isinstance(entry, Prompty):
+            from promptflow._sdk.schemas._flow import ALLOWED_TYPES
+            from promptflow.contracts.tool import ValueType
+            from promptflow.core._model_configuration import PromptyModelConfiguration
+
+            flow_meta = {
+                "inputs": entry._data.get("inputs", {}),
+                "outputs": entry._data.get("outputs", {"output": {"type": "string"}}),
+            }
+            init_dict = {}
+            for field in fields(PromptyModelConfiguration):
+                filed_type = type(field.type).__name__
+                init_dict[field.name] = {"type": filed_type if filed_type in ALLOWED_TYPES else ValueType.OBJECT.value}
+                if field.default != MISSING:
+                    init_dict[field.name]["default"] = field.default
+            flow_meta["init"] = init_dict
+            format_signature_type(flow_meta)
+        elif isinstance(entry, FlexFlow):
+            entry_func = entry_string_to_callable(entry.entry_file, entry.entry)
+            flow_meta, _, _ = FlowOperations._infer_signature_flex_flow(
+                entry=entry_func, language=entry.language, include_primitive_output=True
+            )
+        elif inspect.isclass(entry) or inspect.isfunction(entry):
+            flow_meta, _, _ = FlowOperations._infer_signature_flex_flow(entry=entry, include_primitive_output=True)
```

Comment:
this part is duplicated to origin `_infer_signature`? I would recommend that you directly add 1 `elif` for `Prompty` in origin `_infer_signature`.